client/signup/step-wrapper/index.jsx

```diff
@@ -32,6 +32,7 @@ class StepWrapper extends Component {
 		// Displays an <hr> above the skip button and adds more white space
 		isLargeSkipLayout: PropTypes.bool,
 		isTopButtons: PropTypes.bool,
+		isExternalBackUrl: PropTypes.bool,
```

Comment:
Looks like there's a bit confusion around whether this is `isExternalBackUrl` or `externalBackUrl` :) I'm guessing you renamed it at some point, but looks like there are some usages of the old name in the code.

My vote would go to `isExternalBackUrl`, since it's a boolean.

Reply:
Using only `isExternalBackUrl`now.